recipes/clickhouse-cpp/all/conanfile.py

```diff
@@ -81,15 +63,36 @@ def configure(self):
     def layout(self):
         cmake_layout(self, src_folder="src")
 
+    def requirements(self):
+        self.requires("lz4/1.9.4")
+        self.requires("abseil/20230125.3", transitive_headers=True)
+        self.requires("cityhash/cci.20130801")
+        if self.options.with_openssl:
+            self.requires("openssl/[>=1.1 <4]")
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            check_min_cppstd(self, self._min_cppstd)
+        minimum_version = self._compilers_minimum_version.get(str(self.settings.compiler), False)
+        if minimum_version and Version(self.settings.compiler.version) < minimum_version:
+            raise ConanInvalidConfiguration(f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support.")
+        if self.settings.os == "Windows" and self.options.shared:
+            raise ConanInvalidConfiguration(f"{self.ref} does not support shared library on Windows.")
+            # look at https://github.com/ClickHouse/clickhouse-cpp/pull/226
+
+    def build_requirements(self):
+        if self.options.enable_benchmark:
+            self.requires("benchmark/1.8.2")
+            self.requires("gtest/1.14.0")
```

Comment:
Seems to be broken logic. I guess you want test_requires. But enable_benchmark option should be removed I think.

Reply:
@SpaceIm 
Oh. You are right.
It is not a necessary option.

I will update this PR to remove `enable_benchmark` option and write a comment to the source issue.

Reply:
@Jihadist 
Thanks!